.gitignore

```diff
@@ -18,3 +18,5 @@ reports/*
 !reports/.empty
 
 node_modules/
+
+.DS_Store
```

Comment:
Be careful not to commit OS files like this. I recommend having a `.gitignore` in your home directory for stuff like this:

```
.DS_Store
*.sublime-project
*.sublime-workspace
*.code-workspace
.vscode/
lcov.info
```